base/dataset.go

```diff
@@ -228,9 +228,6 @@ func ListDatasets(ctx context.Context, r repo.Repo, term, profileID string, offs
 			ds.Peername = ref.Peername
 			ds.Name = ref.Name
 			ref.Dataset = ds
-			if RPC {
```

Comment:
I can't remember exactly what, but when converting to HTTP RPC I had to carry this over because removing it produced an odd bug.

Reply:
I *think* the bug was that certain values wouldn't serialize properly over RPC because they didn't have json marshallers defined for their type. I'm thinking of this: https://github.com/qri-io/qri/commit/08322923674f999ad6ebc8e4e822ce4ac03dd291, but possibly for some other type of data. If we do some manual testing and find that it's still a problem, and can't be fixed by defining another serializer hook, we can add something to `scope` that knows if it's operating over RPC. Would be best if we could avoid having the parameter on the input param struct.